.storybook/main.js

```diff
@@ -23,6 +23,17 @@ module.exports = {
       test: /\.stories\.js?$/,
       use: [{ loader: 'story-description-loader' }],
     });
+    config.module.rules.push({
+      test: /\.css?$/,
+      use: [
+        {
+            loader: 'style-loader',
+            options: {
+                injectType: 'lazyStyleTag'
+            }
+        }
```

Comment:
Does the webpack rule also need a `css-loader`?  🤷‍♂️

Reply:
It hasn't had any problems without it 🤷